.travis.yml

```diff
@@ -26,8 +25,7 @@ before_script:
   - composer update
 
 script:
-  - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit ideannotator/tests/; fi
-  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then phpdbg -qrr vendor/bin/phpunit --coverage-clover=coverage.xml; fi
+  - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit ideannotator/tests/ --coverage-clover=coverage.xml; fi
```

Comment:
This will run coverage every time. We remove xdebug in favour of phpdbg because it's more stable with PHP 7

Reply:
Hmmm, never had that problem, but sure enough :)